index.html

```diff
@@ -0,0 +1,205 @@
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <br>
+                    <a class="pro-btn" href="#works-section">See Project</a>
+                </div>
+            </div>
+    
+        <div class="Projects">
+            <div class="Proj-text">
+                <h3>Multi-Post Stories</h3>
+                <h3>Gain+Glory</h3>
+                <ul class="pro-tech">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <br>
+                <a class="pro-btn" href="#works-section">See Project</a>
+            </div>
+        </div>
+    
+        <div class="Projects">
+            <div class="Proj-text">
+                <h3>Multi-Post Stories</h3>
+                <h3>Gain+Glory</h3>
+                <ul class="pro-tech">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <br>
+                <a class="pro-btn" href="#works-section">See Project</a>
+            </div>
+        </div>
+    
+        <div class="Projects">
+            <div class="Proj-text">
+                <h3>Multi-Post Stories</h3>
+                <h3>Gain+Glory</h3>
+                <ul class="pro-tech">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <br>
+                <a class="pro-btn" href="#works-section">See Project</a>
+            </div>
+        </div>
+    
+        <div class="Projects">
+            <div class="Proj-text">
+                <h3>Multi-Post Stories</h3>
+                <h3>Gain+Glory</h3>
+                <ul class="pro-tech">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <br>
+                <a class="pro-btn" href="#works-section">See Project</a>
+            </div>
+        </div>
+    
+        <div class="Projects">
+            <div class="Proj-text">
+                <h3>Multi-Post Stories</h3>
+                <h3>Gain+Glory</h3>
+                <ul class="pro-tech">
+                    <li>Ruby on rails</li>
+                    <li>css</li>
+                    <li>JavaScript</li>
+                    <li>html</li>
+                </ul>
+                <br>
+                <a class="pro-btn" href="#works-section">See Project</a>
+            </div>
+        </div>
+    </section>
+    
+           <section class="about-me">
+            <div id="about-text">
+                <img src="./images/illustration-2-aboutme.svg" alt="image" class="bg-img">
```

Comment:
- [ ] [Required] Please use the image as a `background image` from CSS file instead of `img tag`.